bool[3] charge_leds
bool[3] status_led_color
---
bool success